framework/source/class/qx/ui/layout/Grid.js

```diff
@@ -140,6 +140,21 @@ qx.Class.define("qx.ui.layout.Grid",
       check : "Integer",
       init : 0,
       apply : "_applyLayoutChange"
+    },
+    
+    
+    /**
+     * Allow growing of spanning cells widths beyond the accumulated widths of the columns.
```

Comment:
I think you didn't understand me. This is pedantic, but if you want it correct, change this to:
`Allow growing of spanning cells' widths beyond...`
Note the apostrophe after "cells" to indicate that it is the widths of those cells that is being discussed.

Reply:
OK. OK. Back again.

Reply:
Found the 11 apostrophe rules here : https://www.grammarbook.com/punctuation/apostro.asp
😁

Reply:
Are you loving the English language and all of its inconsistencies, yet? :-)

Reply:
can't wait to see your first German text 😄